py/desispec/exposure_qa.py

```diff
@@ -26,7 +26,7 @@ def get_qa_params() :
     """
     global _qa_params
     if _qa_params is None :
-        param_filename =resource_filename('desispec', 'data/qa/qa-params.yaml')
+        param_filename = resources('desispec').joinpath('data/qa/qa-params.yaml')
```

Comment:
as evidence to not trust the coverage of our unit tests, I'm pretty sure that this should have been `resources.files('desispec')` instead of `resources('desispec')`.  I may need to run a mini-prod with this branch before merging to make sure it fully works.